nncf/tensorflow/quantization/algorithm.py

```diff
@@ -263,6 +264,9 @@ def _parse_range_init_params(self) -> TFRangeInitParams:
     def _parse_group_params(self, quant_config: Dict, quantizer_group: QuantizerGroup) -> None:
         group_name = quantizer_group.value
         params_dict = quant_config.get(group_name, {})
+        if not params_dict and self._target_device != 'VPU':
+            preset = QuantizationPreset.from_str(quant_config.get('preset', 'performance'))
+            params_dict['mode'] = preset.get_quantization_mode(quantizer_group)
```

Comment:
It looks like the preset parameters are a higher priority on the explicitly determined parameters by the user. Just to me, it is not intuitive. Maybe we need to revise this logic or add it to the documentation.

Reply:
Agreed, the explicit parameters should either override preset settings, or there should be an exception thrown prohibiting overrides in case presets are used.

Reply:
Done.